test/e2e/specs/help-center/help-center__general-interaction.ts

```diff
@@ -2,11 +2,141 @@
+			await supportComponent.openPopover();
+		} );
+
+		it( 'Verify Help Center is opened', async function () {
+			expect( await page.locator( '.help-center__container' ).isVisible() ).toBeTruthy();
+		} );
+	} );
+
+	describe( 'Verify Help Center is opened and visible in Editor', function () {
+		it( 'Navigate to the Editor and verify the Help Center is initially closed', async function () {
+			const postURL = DataHelper.getCalypsoURL(
+				'/post/' + testAccount.getSiteURL( { protocol: false } )
+			);
+
+			await page.goto( postURL, {
+				waitUntil: 'networkidle',
+			} );
+		} );
+
+		it( 'Verify Help Center is initially closed', async function () {
+			expect( await page.locator( '.help-center__container' ).isVisible() ).toBeFalsy();
+		} );
+
+		it( 'Open Help Center', async function () {
+			// For Help Center loaded within iframe
+			const helpCenterButtonIframe = page
+				.frameLocator( '.calypsoify iframe' )
+				.getByLabel( 'Help', { exact: true } );
+
+			const helpCenterButtonWithoutIframe = page.locator( 'button.help-center' );
+
+			const helpCenterButton = ( await helpCenterButtonIframe.isVisible() )
+				? helpCenterButtonIframe
+				: helpCenterButtonWithoutIframe;
+
+			await helpCenterButton.click();
+		} );
+
+		it( 'Verify Help Center is opened', async function () {
+			const helpCenterContainerIframe = page
+				.frameLocator( '.calypsoify iframe' )
+				.locator( '.help-center__container' );
+
+			const helpCenterContainerWithoutIframe = page.locator( '.help-center__container' );
+
+			const helpCenterContainer = ( await helpCenterContainerIframe.isVisible() )
+				? helpCenterContainerIframe
+				: helpCenterContainerWithoutIframe;
+
+			const helpCenterContainerIsVisible = await helpCenterContainer.isVisible();
+
+			if ( ! helpCenterContainerIsVisible ) {
+				console.error(
+					`The Help Center is not visible in the editor. ${ helpCenterContainerVisibilityErrorMessage }`
+				);
+			}
+			expect( helpCenterContainerIsVisible ).toBeTruthy();
+		} );
+	} );
+
+	describe( 'Verify Help Center is opened and visible in WP Admin', function () {
+		it( 'Navigate to wp-admin page', async function () {
+			const postURL = `${ testAccount.getSiteURL( {
+				protocol: true,
+			} ) }wp-admin/options-general.php`;
+			await page.goto( postURL, {
+				waitUntil: 'networkidle',
+			} );
+		} );
+
+		it( 'Verify the Help Center is initially closed', async function () {
+			expect( await page.locator( '.help-center__container' ).isVisible() ).toBeFalsy();
+		} );
+
+		it( 'Open Help Center', async function () {
+			await page.locator( '#wp-admin-bar-help-center' ).click();
+		} );
+
+		it( 'Verify Help Center is opened', async function () {
+			const helpCenterContainerIsVisible = await page
+				.locator( '.help-center__container' )
+				.isVisible();
+
+			if ( ! helpCenterContainerIsVisible ) {
+				console.error(
+					`The Help Center is not visible in WP Admin. ${ helpCenterContainerVisibilityErrorMessage }`
+				);
```

Comment:
We should throw an error here instead of logging. This way the message will make it to console.

```suggestion
				throw new Error (
					`The Help Center is not visible in WP Admin. ${ helpCenterContainerVisibilityErrorMessage }`
				);
```